bash_it.sh

```diff
@@ -1,117 +1,129 @@
 #!/usr/bin/env bash
-# Initialize Bash It
-BASH_IT_LOG_PREFIX="core: main: "
 
 # Only set $BASH_IT if it's not already set
-if [ -z "$BASH_IT" ]; then
+if [[ -z "$BASH_IT" ]]; then
 	# Setting $BASH to maintain backwards compatibility
 	export BASH_IT=$BASH
 	BASH="$(bash -c 'echo $BASH')"
 	export BASH
 	BASH_IT_OLD_BASH_SETUP=true
 fi
 
-# Load composure first, so we support function metadata
+# Load composure first, to support function metadata, then create our custom attributes.
 # shellcheck disable=SC1090
 source "${BASH_IT}"/vendor/github.com/erichs/composure/composure.sh
```

Comment:
add prefix and log that we load the file

Reply:
@NoahGorny This is loaded before the logger - no logging can be done yet.

Reply:
I see, can you please add a comment that says exactly what you said here?